pcsx2/gui/MainFrame.cpp

```diff
@@ -636,35 +638,49 @@ void MainEmuFrame::ApplyCoreStatus()
 		}
 	}
 
+	const u8 TotalSources = static_cast<u8>(CDVD_SourceType::NumberOfSources);
+	const CDVD_SourceType Source = g_Conf->CdvdSource;
+	const wxChar* FullBootHelpText[TotalSources];
+	// (X axis - reboot/boot,  Y axis - full/fast,  Z axis - source medium)
+	const wxChar* MenuItemLabels[2][2][TotalSources] = {};
+
+	FullBootHelpText[enum_cast(CDVD_SourceType::Iso)] = _("Boot the VM using the current ISO source media");
+	FullBootHelpText[enum_cast(CDVD_SourceType::Plugin)] = _("Boot the VM using the current CD/DVD source media");
+	FullBootHelpText[enum_cast(CDVD_SourceType::NoDisc)] = _("Boot the VM without any source media");
+
+	MenuItemLabels[0][0][enum_cast(CDVD_SourceType::Iso)]    = _("Boot ISO (f&ull)");
+	MenuItemLabels[0][1][enum_cast(CDVD_SourceType::Iso)]    = _("Boot ISO (&fast)");
+	MenuItemLabels[1][0][enum_cast(CDVD_SourceType::Iso)]    = _("Reboot ISO (f&ull)");
+	MenuItemLabels[1][1][enum_cast(CDVD_SourceType::Iso)]    = _("Reboot ISO (&fast)");
+	MenuItemLabels[0][0][enum_cast(CDVD_SourceType::Plugin)] = _("Boot CDVD (f&ull)");
+	MenuItemLabels[0][1][enum_cast(CDVD_SourceType::Plugin)] = _("Boot CDVD (&fast)");
+	MenuItemLabels[1][0][enum_cast(CDVD_SourceType::Plugin)] = _("Reboot CDVD (f&ull)");
+	MenuItemLabels[1][1][enum_cast(CDVD_SourceType::Plugin)] = _("Reboot CDVD (&fast)");
+	MenuItemLabels[0][0][enum_cast(CDVD_SourceType::NoDisc)] = _("Boot BIOS");
+	MenuItemLabels[1][0][enum_cast(CDVD_SourceType::NoDisc)] = _("Reboot BIOS");
```

Comment:
Well, it's not optimal, and I think using switches and ifs would be nicer and have the additional benefit of not having to enum cast every single thing.